### PR TITLE
[requires.io] dependency update on develop branch

### PR DESCRIPTION
**Pull request by maurizi**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-boto==2.38.0
+boto==2.39.0
```
Comment (maurizi) [inline requirements.txt]: Release notes for boto 2.39.0: http://docs.pythonboto.org/en/latest/releasenotes/v2.39.0.html

Seems like a safe upgrade

File: requirements.txt [hunk 2]
```diff
@@ -21,7 +21,7 @@ git+https://github.com/dahlia/libsass-python.git@4aa8fd3c2cef8c1
 django-contrib-comments==1.6.2
 django-threadedcomments==1.0.1
 django-apptemplates==1.0
-django-queryset-csv==0.3.2
+django-queryset-csv==0.3.3
```
Comment (maurizi) [inline requirements.txt]: @steventlamb What do you think, can we safely upgrade this?

Comment (steventlamb) [inline requirements.txt]: :+1:

File: test-requirements.txt [hunk 3]
```diff
@@ -1,3 +1,3 @@
 PyVirtualDisplay==0.1.5
-selenium==2.49.0
+selenium==2.49.2
```
Comment (maurizi) [inline test-requirements.txt]: The UI tests passed, so this seems like a safe upgrade to me.

**Comment by RickMohr** [comment]
What level of review do you think is warranted? Reprovision and run tests plus look at the release notes?

**Comment by maurizi** [comment]
I already took a look at the release notes and left my thoughts, so I don't think you need to do that as well (unless you want to).

I think reprovisioning + tests should be fine

File: test-requirements.txt (hunk 3 above)
Comment (RickMohr) [inline test-requirements.txt]: I'm seeing failures like "TypeError: string indices must be integers", which (from my reading of https://github.com/SeleniumHQ/selenium/issues/1497) appears to have been introduced in 2.49.1, is present in 2.50, and was fixed only three days ago.

On the `develop` branch I see "No JSON object could be decoded" errors, which (from my reading of https://github.com/SeleniumHQ/selenium/issues/1478) were introduced in 2.49.0

All of which begs the question of why the UI tests succeed for you.

Comment (maurizi) [inline test-requirements.txt]: Odd - [they passed on CircleCI](https://circleci.com/gh/OpenTreeMap/otm-core/837), I didn't run them locally.

I checked and CircleCI _did_ install 2.49.2 of selenium....

Comment (maurizi) [inline test-requirements.txt]: Also we're running `2.49.0`, so I don't know why we would be encountering "No JSON object could be decoded" already if it was introduced in `2.49.0`... very odd.

**Comment by RickMohr** [comment]
Summary of changes

Module | Old | New | Changes | Comments
------ | --- | --- | ----- | --------
boto | 2.38.0 | 2.42.0 | [changes](http://docs.pythonboto.org/en/latest/releasenotes/v2.39.0.html)
coverage | 4.0.3 | 4.2 | [changes](https://coverage.readthedocs.io/en/coverage-4.2/changes.html)
Django | 1.8.8 | 1.8.14 | [changes](https://docs.djangoproject.com/en/1.10/releases/#id2)
django-apptemplates | 1.0 | 1.2 | [changes](https://pypi.python.org/pypi/django-apptemplates/)
django-contrib-comments | 1.6.2 | 1.7.3 | [changes](https://pypi.python.org/pypi/django-contrib-comments)
django-debug-toolbar | 1.4 | 1.5 | [changes](https://django-debug-toolbar.readthedocs.io/en/stable/changes.html)
django-extensions | 1.6.1 | 1.7.4 | [changes](https://github.com/django-extensions/django-extensions/blob/master/CHANGELOG.md)
django-hstore | 1.4.1 | 1.4.2 | [changes](https://pypi.python.org/pypi/django-hstore/)
django-queryset-csv | 0.3.2 | 0.3.3 | [commits](https://github.com/azavea/django-queryset-csv/commits/master)
django-redis | 4.3.0 | 4.4.4 | [changes](https://github.com/niwinz/django-redis/blob/master/CHANGES.txt)
django-registration-redux | 1.3 | 1.4 | [changes](https://django-registration-redux.readthedocs.io/en/latest/upgrade.html)
django-storages | 1.1.8 | 1.5.1 | [changes](https://pypi.python.org/pypi/django-storages)
django-webpack-loader | 0.3.0 | 0.3.3 | [commits](https://github.com/owais/django-webpack-loader/commits/master)
gunicorn | 19.4.5 | 19.6.0 | [changes](http://docs.gunicorn.org/en/stable/news.html)
Pillow | 3.1.0 | 3.3.1 | [changes](https://github.com/python-pillow/Pillow/blob/master/CHANGES.rst)
psycopg2 | 2.6.1 | 2.6.2 | [changes](http://initd.org/psycopg/docs/news.html)
PyVirtualDisplay | 0.1.5 | 0.2 | [commits](https://github.com/ponty/PyVirtualDisplay/commits/master)
python-dateutil | 2.4.2 | 2.5.3 | [changes](https://github.com/dateutil/dateutil/blob/master/NEWS)
python-omgeo | 1.7.2 | 1.9.0 | [changes](https://github.com/azavea/python-omgeo/blob/master/CHANGES.rst)
pytz | 2015.7 | 2016.6.1 | [changes](https://github.com/newvem/pytz/blob/master/CHANGES.txt)
rollbar | 0.11.1 | 0.13.3 | [changes](https://github.com/rollbar/pyrollbar/blob/master/CHANGELOG.md)
selenium | 2.49.0 | 2.53.6 | [changes](https://github.com/SeleniumHQ/selenium/blob/master/py/CHANGES)

**Comment by RickMohr** [comment]
Closed in favor of #2780

**Pull request closed without merging**